topo_processor/metadata/lds_cache/lds_cache.py

```diff
@@ -52,10 +55,24 @@ def get_metadata(
             if os.path.isfile(metadata_path):
                 if exported_asset.extra_fields.get("encoding", None) == "gzip":
                     decompress_file(metadata_path)
+                    new_metadata_path = os.path.splitext(metadata_path)[0] + "_" + time.strftime("%s") + ".csv"
+                    geopackage_to_csv(metadata_path, new_metadata_path).run()
+                    metadata_path = new_metadata_path
             else:
                 raise Exception(f"{metadata_path} not found")
 
     if os.path.isfile(metadata_path):
+        if is_geopackage(metadata_path):
```

Comment:
There is a code duplication you could get rid of by having
```
if os.path.isfile(metadata_path):
 if exported_asset.extra_fields.get("encoding", None) == "gzip":
   decompress_file(metadata_path)
 new_metadata_path = os.path.splitext(metadata_path)[0] + "_" + time.strftime("%s") + ".csv"
 geopackage_to_csv(metadata_path, new_metadata_path).run()
 metadata_path = new_metadata_path
```
only once

Reply:
Oops my bad that was a mistake.